src/dotnet/CarbonAware.Tools.WattTimeClient/Constants/QueryStrings.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace CarbonAware.Tools.WattTimeClient.Constants;
+
+internal class QueryStrings
+{
+    public const string BalancingAuthorityAbbreviation = "ba";
+    public const string StartTime = "starttime";
+    public const string EndTime = "endtime";
+    public const string Latitude = "latitude";
+    public const string Longitude = "longitude";
+    public const string Username = "username";
```

Comment:
>Username [](http://example.com/codeflow?start=24&length=8)

from the docs, it doesn't appear we need this, since as a QueryString it is only supported on the Password Reset API ... ?

[https://www.watttime.org/api-documentation/#password-reset](https://www.watttime.org/api-documentation/#password-reset)

Reply:
We are also using these querystring constants to tag the tracing objects with relevant params, so it best fit with this group of constants even though we don't use it as a technical query string, we do use it to label the username param.